starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,55 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  background-image: url("../images/apple-pie.jpg");
+  background-position: 0 0;
+  background-repeat: no-repeat;
+  background-size: 100%;
+  height: 100vh ;
+}
+
+h1 {
+  display: flex;
```

Comment:
Using Flexbox for simply centering a heading is slightly overkill. You usually use Flexbox on containers to arrange elements that are inside.
Here, you could have simply used
```css
text-align: center;
```
on the h1.